modules/eventing/pages/eventing-rbac.adoc

```diff
@@ -0,0 +1,141 @@
+** bucket.scope or bucket.*
+
+== Minimal Eventing RBAC role example
+
+* Access the Couchbase Web Console > Security and Select "ADD GROUP".
++
+image::rbac_min_add_add_group.png[,%100]
+
+* Configure the group as follows "Data Reader" and "Data Writer" are required for the Eventing Storage or scratchpad. "Data DCP Reader" is required to fetch the mutations form DCP. Note this item was defined as 'bulk.data.*' which would allow building Evening functions that can listen to any collection under 'bulk.data'.
++
+image::rbac_min_a.png[,538,align=middle]
++
+The final item required is defining the *Function Scope* under "Eventing / Manage Scope Function" since we will be listing to mutations in a collection under 'bulk.data' it makes sense to use this as our grouping.
++
+image::rbac_min_b.png[,538,align=middle]
+
+* Hit *Save* to store the GROUP to the system.
+
+* Access the Couchbase Web Console > Security and Select "ADD USER".
++
+image::rbac_min_add_add_user.png[,%100]
+
+* Associate the GROUP to the user so the user can inherit all the roles in the group.
++
+image::rbac_min_c.png[,538,align=middle]
+
+* Add your password and verify it in the lower two boxes
+
+* Hit *Save* to store the USER to the system.
+
+* Access the Couchbase Web Console > Security 
+
+* Select GROUPS on the right, you should see your definition for GROUP "eventing_min"
++
+image::rbac_min_groups.png[,%100]
+
+* Select USERS on the right, you should see your definition for USER "user_min"
++
+image::rbac_min_users.png[,%100]
+
+== Beyond a Minimal Eventing RBAC role
+
+You may consider adding
+
+* Data Reader
+** Mutation Source
+* Data Writer
+** Mutation Source
+* Data Monitor
+** Mutation Source
+** Eventing Storage keyspace or Scratchpad
+
+If you have any Bindings in your Eventing Function of type "Bucket Alias" you will need to have one or more additional settings if not already allowed.
+
+* Data Reader
+** Bucket Alias
+* Data Writer
+** Bucket Alias
+
+If you plan to use N1QL consider adding at lease SELECT privileges
+
+* Query & Index / Query Select
+** Mutation Source
+
+== Multi-tenancy in Eventing
+
+The "Function Scope" in an Eventing Function works with the RBAC selection in "Eventing / Manage Scope Function" to limit access to between tenants in both the UI and the REST API. 
+
+A tenant might be based on company departments such as administration, sales, production and support.
+
+Below we have two tenants example (an admin and a limited user) and four Eventing Functions each with a different *Function Scope*, we logged into the UI with either an Eventing Full Admin" or "Full Admin" role and thus we can access all of the  Eventing Functions. 
+
+image::rbac_admin_view.png[,%100]
+
+Now log out of the UI console and log back in as a non-privileged user (for example we use the USER "user_min" as defined above).  Because of the privliges defined we are only allowed access to Eventing Functions that have a *Function Scope* of 'bulk.data'.
```

Comment:
```suggestion
Now log out of the UI console and log back in as a non-privileged user (for example we use the USER "user_min" as defined above).  
Because of the privliges defined we are only allowed access to Eventing Functions that have a *Function Scope* of 'bulk.data'.
```